### PR TITLE
support loggingPrefs capability for chrome sessions

### PR DESCRIPTION
**Pull request by jlipps**
we did not previously support the `loggingPrefs` capability for chrome sessions.

### CODE REVIEW

File: lib/commands/context.js [hunk 1]
```diff
@@ -310,8 +310,18 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
...
+    caps.loggingPrefs = opts.loggingPrefs;
+  }
   if (opts.enablePerformanceLogging) {
-    caps.loggingPrefs = {performance: 'ALL'};
+    log.warn(`The 'enablePerformanceLogging' cap is deprecated; simply use ` +
+             `the 'loggingPrefs' cap instead, with a 'performance' key set to 'ALL'`);
+    const newPref = {performance: 'ALL'};
+
+    // don't overwrite other logging prefs that have been sent in if they exist
+    caps.loggingPrefs = caps.loggingPrefs ?
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: `caps.loggingPrefs = opts.loggingPrefs || {performance: 'ALL'}`

File: lib/commands/context.js [hunk 2]
```diff
@@ -310,8 +310,18 @@ helpers.setupNewChromedriver = async function setupNewChromedriver (opts, curDev
...
+      Object.extend({}, caps.loggingPrefs, newPref) :
```
Comment (mykola-mokhnach) [inline lib/commands/context.js]: perhaps `Object.assign`?

**Comment by jlipps** [comment]
cool, i merged this, however i don't have the `ant` trickery set up on my machine in order to publish this; @imurchie do you mind doing so?

**Comment by imurchie** [comment]
@jlipps `appium-android-driver@4.4.0` published.

**Comment by jlipps** [comment]
thanks!

**Pull request merged**